fastlane/fastlane.gemspec

```diff
@@ -3,6 +3,12 @@ lib = File.expand_path('../lib', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'fastlane/version'
 
+# Copy over the latest .rubocop.yml style guide
+# This will use the file name without the `.` as 
+# hidden files are ignored
+rubocop_config = File.expand_path('../../.rubocop.yml', __FILE__)
+`cp #{rubocop_config} #{lib}/fastlane/plugins/template/rubocop.yml`
```

Comment:
Evaluating this as part of fastlane's gemspec feels weird to me. Can we talk about why this is necessary?